src/sous_chef/templates/base.html

```diff
@@ -70,6 +70,12 @@
     {% endif %}
     <!-- WARNING :  Do not move this line before a manual import of script! -->
     {% block extrajs %}{% endblock %}
+
+    <script type="application/javascript">
+      $(function () {
+        for(var f=document.forms,i=f.length;i--;)f[i].setAttribute("novalidate",i)
+      });  // disable HTML5 form validation
+    </script>
```

Comment:
@lingxiaoyang I think is an elegant solution to the problem, but I don't like the idea of not taking advantage of the html5 validation in the all other fields where the validation works as expected. 
Why not try to identify the fields where the html5 validation doesn't work as expected and remove the "required" attribute or the one that forces the validation?
I have tested a small modification to your change in the client creation form and I can have html5 validations in all the other fields except in the "route", for this one we can relay in the server validation.
`for(var f=document.querySelectorAll('select'),i=f.length;i--;)f[i].removeAttribute("required",i)`